### DISCUSSION TITLE
Google Maps Oops!

### DISCUSSION THREAD

**Discussion by DCO84770** [code4recovery/12-step-meeting-list#738] [Q&A] [answered]
Hello - I'm in the process of connecting our site to the Meeting App. I have one test meeting that shows up on our meetings page, but I get an error when clicking on MAP: "This page didn't load Google Maps correctly. "

**Comment by tech2serve** [comment, marked as answer]
Hi @DCO84770, thanks for opening a Discussion.

Most likely you haven't set up a mapping key correctly (Google or MapBox). If you need more help, please search the Discussions for more examples.

We recommend using MapBox vice Google as it is easier, doesn't require a credit card, and will be more compatible with TSML UI (our future replacement for TSML's public meeting list).

**Reply by DCO84770** [reply]
Thanks. I'll look up MapBox
